src/renderer/components/drawer/drawer.tsx

```diff
@@ -137,7 +144,10 @@ export class Drawer extends React.Component<DrawerProps> {
         <div className={className} style={style} ref={e => this.contentElem = e}>
           <div className="drawer-wrapper flex column">
             <div className="drawer-title flex align-center">
-              <div className="drawer-title-text">{title}</div>
+              <div className="drawer-title-text">
```

Comment:
Add some extra class names to make sure copy icon will be placed correctly with some super-long object name


```
<div className="drawer-title-text flex gaps align-center">
```